src/users/usersActions.js

```diff
@@ -35,6 +35,76 @@ export const reportPresence = (hasFocus: boolean = true, newUserInput: boolean =
   });
 };
 
+const typingWorkerBase = auth => ({
```

Comment:
It seems a bit odd to call this a `Base` when it's not being used for inheritance.

(It _is_ being used for Self-style inheritance, though; I suppose that's close enough. :upside_down_face:)

Reply:
> (It _is_ being used for Self-style inheritance, though; I suppose that's close enough. )

That's more or less what I was thinking :slightly_smiling_face: , though I didn't have Self's name specifically in mind.

I was thinking more like "base layer" or "base model" than object-oriented inheritance... but it's not really a coincidence that the same word appears there too.